ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -101,20 +106,27 @@ if tflite_support_is_available
   nnstreamer_filter_tflite_deps = tflite_support_deps + [thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
 
   tflite_compile_args = []
+  tflite_compile_args += '-DTFLITE_VERSION=1'
```

Comment:
Discussion
TFLITE_VERSION='1.13.1'
TFLITE_MAJOR_VERSION=1
TFLITE_MINOR_VERSION=13
TFLITE_MICRO_VERSION=1

how about changing all version definition?

Reply:
OK, I'll update the PR.

Reply:
There is ```'-DTFLITE_VER``` in ext/nnstreamer/tensor_filter/meson.build.
```python
    tflite_ver_dep = declare_dependency (
      compile_args : ['-DTFLITE_VER="' + tflite_support_deps[0].version() +'"', ],
    )
```

How about generalizing this compile_args?

Reply:
Oh, I didn't see that. I will reuse or revise this.